.solcover.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   skipFiles: [
     'Migrations.sol',
     'mocks',
-    'external'
+    'external',
+    'core/RebalancingTokenFactory.sol'
```

Comment:
Ignoring this test suite for now for now. It needs to be tested via core since the transaction sender needs to be an account.